uint16 number
---
float32[3] centerPoint
